### ISSUE TITLE
Menu for copy code to clipboard.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google/code-prettify#318]
```
I had a old prettivy code that placed above the code a menu bar so you could 
copy the code to the clipboard , is that gone?

See attachment.
```

Original issue reported on code.google.com by `Pascal.B...@googlemail.com` on 10 Nov 2013 at 8:34

Attachments:
- [bar.png](https://storage.googleapis.com/google-code-attachments/google-code-prettify/issue-318/comment-0/bar.png)

**Comment by amroamroamro** [comment]
I don't think this was ever in google-code-prettify, you must have been using another library.

Of course you can always do this yourself with a bit of Javascript to add such a menu to every pre/code blocks.
